db/database.go

```diff
@@ -4,17 +4,20 @@ import (
 	"encoding/json"
 
 	"github.com/athanorlabs/atomic-swap/common/types"
+	"github.com/athanorlabs/atomic-swap/protocol/swap"
 
 	"github.com/ChainSafe/chaindb"
 )
 
-var (
+const (
 	offerPrefix = "offer"
+	swapPrefix  = "swap"
 )
 
 // Database is the persistent datastore used by swapd.
```

Comment:
Can we document what the key/value pairs are for the 2 databases here?

Reply:
Reading down further in the PR, we'd also want to mention when `swapTable` entries are updated.

Reply:
comment added, let me know what you think